raft/testdata/confchange_v2_add_double_auto.txt

```diff
@@ -195,3 +195,213 @@ stabilize 1 3
+  1/8 EntryNormal "bar"
+  CommittedEntries:
+  1/6 EntryConfChangeV2 r2 r3
+  Messages:
+  3->1 MsgAppResp Term:1 Log:0/7
+  3->1 MsgAppResp Term:1 Log:0/8
+  3->1 MsgAppResp Term:1 Log:0/8
+  INFO 3 switched to configuration voters=(1)&&(1 2 3) autoleave
+
+# n2 and n3 also leave the joint config and the dust settles. We see at the very
+# end that n1 receives some messages from them that it refuses because it does
+# not have them in its config any more.
+stabilize
+----
+> 1 receiving messages
+  2->1 MsgAppResp Term:1 Log:0/7
+  2->1 MsgAppResp Term:1 Log:0/8
+  2->1 MsgAppResp Term:1 Log:0/8
+  3->1 MsgAppResp Term:1 Log:0/7
+  3->1 MsgAppResp Term:1 Log:0/8
+  3->1 MsgAppResp Term:1 Log:0/8
+> 1 handling Ready
+  Ready MustSync=false:
+  HardState Term:1 Vote:1 Commit:8
+  CommittedEntries:
+  1/7 EntryNormal "foo"
+  1/8 EntryNormal "bar"
+  Messages:
+  1->2 MsgApp Term:1 Log:1/8 Commit:7 Entries:[1/9 EntryConfChangeV2]
+  1->3 MsgApp Term:1 Log:1/8 Commit:7 Entries:[1/9 EntryConfChangeV2]
+  1->2 MsgApp Term:1 Log:1/9 Commit:8
+  1->3 MsgApp Term:1 Log:1/9 Commit:8
+> 2 receiving messages
+  1->2 MsgApp Term:1 Log:1/8 Commit:7 Entries:[1/9 EntryConfChangeV2]
+  1->2 MsgApp Term:1 Log:1/9 Commit:8
+> 3 receiving messages
+  1->3 MsgApp Term:1 Log:1/8 Commit:7 Entries:[1/9 EntryConfChangeV2]
+  1->3 MsgApp Term:1 Log:1/9 Commit:8
+> 2 handling Ready
+  Ready MustSync=true:
+  HardState Term:1 Commit:8
+  Entries:
+  1/9 EntryConfChangeV2
+  CommittedEntries:
+  1/7 EntryNormal "foo"
+  1/8 EntryNormal "bar"
+  Messages:
+  2->1 MsgAppResp Term:1 Log:0/9
+  2->1 MsgAppResp Term:1 Log:0/9
+> 3 handling Ready
+  Ready MustSync=true:
+  HardState Term:1 Commit:8
+  Entries:
+  1/9 EntryConfChangeV2
+  CommittedEntries:
+  1/7 EntryNormal "foo"
+  1/8 EntryNormal "bar"
+  Messages:
+  3->1 MsgAppResp Term:1 Log:0/9
+  3->1 MsgAppResp Term:1 Log:0/9
+> 1 receiving messages
+  2->1 MsgAppResp Term:1 Log:0/9
+  2->1 MsgAppResp Term:1 Log:0/9
+  3->1 MsgAppResp Term:1 Log:0/9
+  3->1 MsgAppResp Term:1 Log:0/9
+> 1 handling Ready
+  Ready MustSync=false:
+  HardState Term:1 Vote:1 Commit:9
+  CommittedEntries:
+  1/9 EntryConfChangeV2
+  Messages:
+  1->2 MsgApp Term:1 Log:1/9 Commit:9
+  1->3 MsgApp Term:1 Log:1/9 Commit:9
+  INFO 1 switched to configuration voters=(1)
+> 2 receiving messages
+  1->2 MsgApp Term:1 Log:1/9 Commit:9
+> 3 receiving messages
+  1->3 MsgApp Term:1 Log:1/9 Commit:9
+> 2 handling Ready
+  Ready MustSync=false:
+  HardState Term:1 Commit:9
+  CommittedEntries:
+  1/9 EntryConfChangeV2
+  Messages:
+  2->1 MsgAppResp Term:1 Log:0/9
+  INFO 2 switched to configuration voters=(1)
+> 3 handling Ready
+  Ready MustSync=false:
+  HardState Term:1 Commit:9
+  CommittedEntries:
+  1/9 EntryConfChangeV2
+  Messages:
+  3->1 MsgAppResp Term:1 Log:0/9
+  INFO 3 switched to configuration voters=(1)
+> 1 receiving messages
+  2->1 MsgAppResp Term:1 Log:0/9
+  raft: cannot step as peer not found3->1 MsgAppResp Term:1 Log:0/9
```

Comment:
Is there something we can do here to add a new-line after the `ErrStepPeerNotFound` error?